use-cases/single-file-C-program/test-main.c

```diff
@@ -103,7 +113,7 @@ test_perf_fast_logging(int nMil)
     }
     uint32_t n = 0;
     for (n = 0; n < (nMil * L3_MILLION); n++) {
-        l3_log_fast("300-Mil Fast l3-log msgs", n, 0);
+        l3_log_fast("Perf-300-Mil Fast l3-log msgs", n, 0);
```

Comment:
We do not want to grep for these instances and on L93, above, so prefix the message with the term `Perf-`.

The grep code in `build_exp_loc_msg_list()` will filter these instances out.